src/test/Components/CreateImageWizardV2/CreateImageWizard.content.test.tsx

```diff
@@ -475,36 +475,9 @@ describe('Step Custom repositories', () => {
       })
     );
 
-    await screen.findByText(/select all \(1016 items\)/i);
+    await screen.findByText(/select page \(10 items\)/i);
```

Comment:
I've discussed with @lucasgarfield previously that testing on 1000 of mocked repositories might be a bit too much. Maybe we could also update the fixture to generate less filler repos? Just a though, might not be completely in the scope of this PR :sweat_smile:

Reply:
Agree, let's stick with 10 here. The fixtures can be updated later if needed.